iaso/api/instances.py

```diff
@@ -598,6 +620,68 @@ def bulkdelete(self, request):
             status=201,
         )
 
+    @action(
+        detail=False,
+        methods=["GET"],
+        permission_classes=[permissions.IsAuthenticated, HasInstanceBulkPermission, HasCreateOrgUnitPermission],
+    )
+    def check_bulk_gps_push(self, request):
+        # first, let's parse all parameters received from the URL
+        select_all, selected_ids, unselected_ids = self._parse_check_bulk_gps_push_parameters(request.GET)
```

Comment:
You should probably use a serializer instead of manually parsing the querystring?